client/app/services/service-details/service-details.html

```diff
@@ -10,24 +10,32 @@
 </div>
 <pf-toolbar class="section-toolbar section-toolbar-right-actions" config="vm.headerConfig"
             ng-if="!vm.loading">
-    <actions>
-        <div class="ss-details-header__actions">
-            <div uib-dropdown class="ss-details-header__actions__inner dropdown-kebab-pf">
-                <custom-button ng-if="vm.hasCustomButtons(vm.service)"
-                               service-id="vm.service.id"
-                               custom-actions="vm.service.custom_actions">
-                </custom-button>
-                <custom-dropdown class="custom-dropdown pull-left"
-                                 config="item"
-                                 items="vm.selectedItemsList"
-                                 items-count="vm.selectedItemsListCount"
-                                 ng-repeat="item in vm.listActions"
-                                 on-update="vm.listActionDisable($config, $changes)"
-                                 menu-right="true">
-                </custom-dropdown>
-            </div>
-        </div>
-    </actions>
+  <actions>
+    <div class="ss-details-header__actions">
+      <div uib-dropdown class="ss-details-header__actions__inner dropdown-kebab-pf">
+        <custom-button
+          ng-if="vm.hasCustomButtons(vm.service) <= 3 && vm.hasCustomButtons(vm.service) > 0"
+          class="custom-dropdown pull-left"
+          service-id="vm.vmDetails.service.id"
```

Comment:
This is a surprise.. why is `vmDetails` needed with 1-3 buttons but not before or with 4+?

Reply:
yep, woops, that one slipped through.. 

bad copypasta indeeeeed 😋